sources/packages/backend/libs/services/src/notifications/notification/notification-actions.service.ts

```diff
@@ -1258,6 +1259,44 @@ export class NotificationActionsService {
     return { templateId, emailContacts };
   }
 
+  /**
+   * Creates student application notification for student.
```

Comment:
Please be specific about the notification. There will other application notifications.